cd/scripts/release-controller.sh

```diff
@@ -122,21 +121,42 @@ popd 1>/dev/null
 # build controller image
 if ! buildah bud \
   --quiet="$QUIET" \
-  -t "$AWS_SERVICE_DOCKER_IMG" \
+  -t "$AWS_SERVICE_DOCKER_IMG"-amd64 \
   -f "$CONTROLLER_IMAGE_DOCKERFILE_PATH" \
   --build-arg service_alias="$AWS_SERVICE" \
   --build-arg service_controller_git_version="$CHART_VERSION" \
   --build-arg service_controller_git_commit="$SERVICE_CONTROLLER_GIT_COMMIT" \
   --build-arg build_date="$BUILD_DATE" \
   --build-arg golang_version="$GOLANG_VERSION" \
-  --build-arg go_arch="$GOARCH" \
+  --build-arg go_arch="linux/amd64" \
```

Comment:
To keep consistant with the changes in https://github.com/aws-controllers-k8s/code-generator/pull/461/files we need to name these `--build-arg target_arch="linux/amd64"

Reply:
As soon as the content of this conversation is determined, we will re-string it with that argument name.
Thank you!
https://github.com/aws-controllers-k8s/code-generator/pull/461#discussion_r1337906559

Reply:
done.